### PR TITLE
fix minor typos and formatted text

### PR DESCRIPTION
**Pull request by victorhck**
### PR creator: Description

Describe the overall goals of this pull request.

Fix minor typos and solve wrong formatted text

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [X] SLE 15 SP3/openSUSE Leap 15.3
  - [X] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/opensuse_update.xml [hunk 1]
```diff
@@ -419,8 +419,8 @@ udev           252M  124K  252M   1% /dev
        <listitem>
         <para>
          This section allows you to change the <guimenu>Primary
-         Language</guimenu>primary language and configure additional
-         <guimenu>Secontry Languages.</guimenu>. Optionally, you can adjust the
+         Language</guimenu> and configure additional
+         <guimenu>Secontry Languages.</guimenu> Optionally, you can adjust the
```
Comment (cjschroder) [inline xml/opensuse_update.xml:423]: Hi @victorhck, thank you for the PR! "Secontry Languages" looks like a misspelling, should that be "Secondary Languages"?

**Review by cjschroder — CHANGES_REQUESTED** [review]

File: xml/opensuse_update.xml (hunk 1 above)
Comment (victorhck) [reply xml/opensuse_update.xml:423]: ups! I though it was a right term, that's why I kept the term...

**Comment by victorhck** [comment]
updated the PR. Fixed the misspelling word

File: xml/opensuse_update.xml (hunk 1 above)
Comment (cjschroder) [reply xml/opensuse_update.xml]: I know, right? It could have been real :)  Thanks for fixing it.

**Comment by cjschroder** [comment]
SLE15SP3 edca930c4 be4895def
SLE15SP2 0b2e1df4b 7ac62f721

**Pull request merged**